.github/workflows/rust.yml

```
@@ -66,6 +66,24 @@ jobs:
       - name: Test docs
         run: cargo test --doc --profile ci --features intl
 
+  msrv:
+    name: Minimum supported Rust version
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    steps:
+      - uses: actions/checkout@v4
+      # Get the rust_version from the Cargo.toml
+      - name: Get rust_version
+        id: rust_version
+        run: echo "::set-output name=rust_version::$(grep '^rust-version' Cargo.toml | cut -d' ' -f3 | tr -d '"')"
```

Huh, looks like `set-output` is going to be [deprecated soon](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/), so maybe something more akin to below.

```suggestion
        run: echo "rust_version=$(grep '^rust-version' Cargo.toml | cut -d' ' -f3 | tr -d '"')" >> $GITHUB_OUTPUT
```